File: python/Warmup-2/last2.py
def last2(str):
	if len(str) < 2:
		return 0

	count = 0 
	end = str[-2:]

	for i in range(len(str) - 2):
		substring = str[i:i + 2]
	
		if substring == end:
			count += 1

	return count

File: java/Warmup-2/last2.java
public int last2(String str) {
	int count = 0;

	if (str.length() > 1) {

		String end = str.substring(str.length() - 2);
		for (int i = 0; i < str.length() - 2; i++) {
			if (str.substring(i, i+2).equals(end)) count++;
		}
	}

	return count;
}

